autopilot/agent_test.go

```diff
@@ -372,77 +377,24 @@ func TestAgentChannelFailureSignal(t *testing.T) {
 		Score:  0.5,
 	}
 
-	select {
-	case heuristic.nodeScoresResps <- map[NodeID]*NodeScore{
-		NewNodeID(node): fakeDirective,
-	}:
-	case <-time.After(time.Second * 10):
-		t.Fatal("heuristic wasn't queried in time")
-	}
+	respondNodeScores(t, testCtx,
```

Comment:
Nit: should be of the form
```
respondNodeScores(
    ..., ...,
)
```

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/2633/commits/28f79b54069838fdabf89f069214cd203ed6c4bc